04-graphs/07-graphs-activity.md

```diff
@@ -6,27 +6,23 @@ We will go over how to implement a solution for the Bipartition Graph problem.
 
 ### Class Livecode
 
-Follow along in the following Replit as we livecode the function in class: 
-* [Bipartition Graph Livecode Start](https://replit.com/@adadev/Bipartition-Graph-Livecode-Start)
-* [C19 Bipartition Graph Livecode Solution](https://replit.com/@adadev/Bipartition-Graph-Livecode-C19)
-* [Bipartition Graph Expanded Solution](https://replit.com/@adadev/Bipartition-Graph-Instructor-Solution)
+Follow along in the following repo as we livecode the function in class: 
+* [Bipartition Graph Livecode Start](https://github.com/Ada-Activities/bipartition_graph_livecode)
+  * Check out the `solution` branch in the repo to see a version of the Livecode implementation
+
 
 ### Small Group Work: Graph Variations Activities
 
 For your small group activity,  you will work on one of the graph variation activities.
 
 - Spend twenty minutes with your breakout room implementing a solution. 
-- One person in the group should fork the replit and share a link with the rest of the group.
+- One person in the group should fork the repo and share a link with the rest of the group.
 - If your entire group is feeling stuck, start by discussing how you would solve the function using pen and paper.
 - If your entire group is still feeling lost, discuss what is making you feel stuck and ask an instructor to come to your room for help!
 - After small group work, we will come back together as a class to work through a solution together, as time allows. Groups may ask questions, share where they got stuck, and/or share out their solutions.
 
-Fork one of the following Replits to work on in small groups: 
-* [Convert Graph](https://replit.com/@adadev/Convert-Graph-Activity)
-* [DFS with Adjacency Matrix](https://replit.com/@adadev/dfs-with-adjacency-matrix)
-* [Repeat BFS](https://replit.com/@adadev/Repeat-BFS-Activity)
-
-Solutions:
-* [Convert Graph](https://replit.com/@adadev/Convert-Graph-Activity-Solution)
-* [DFS with Adjacency Matrix](https://replit.com/@adadev/dfs-with-adjacency-matrix-solution)
-* [Repeat BFS](https://replit.com/@adadev/Repeat-BFS-Activity-Solution)
+Fork one of the following repos to work on in small groups: 
+* [Convert Graph](https://github.com/Ada-Activities/convert_graph_activity)
+* [DFS with Adjacency Matrix](https://github.com/Ada-Activities/dfs-with-adjacency-matrix)
+* [Repeat BFS](https://github.com/Ada-Activities/repeat-bfs-activity/)
+  * Check out the `solution` branch in each repo to see a version of a working implementation
```

Comment:
Even though it mentions each repo, it makes it look like it only applies to Repeat BFS. My thought would be to have it as a standalone line beneath the repos.
```suggestion

Check out the `solution` branch in each repo to see a version of a working implementation
```

Reply:
Great catch to prevent confusion, thank you!